RedHatAI/Kimi-K2-Instruct-quantized.w4a16/performance/server.yml

```diff
@@ -0,0 +1,5 @@
+enable-chunked-prefill: true

```

Comment:
couple of these parameters are already in common/performance. Just curious what the difference is with common vs per model args?